domains/xhide341.json

```
@@ -0,0 +1,12 @@
+{
+    "description": "Hey, I'm hide and I'm a developer",
+    "repo": "https://github.com/xhide341/portfolio",
+    "owner": {
+        "username": "xhide341",
+        "email": "shawnehgn10@gmail.com",
+        "discord": "364749406878171136"
+    },
+    "record": {
+        "CNAME": ["xhide341.netlify.app"]
```

CNAME value should not be in an array
```suggestion
        "CNAME": "xhide341.netlify.app"
```

---

forgot the array, fixed it now thanks.